test/unittests/state_transition.cpp

```diff
@@ -225,7 +227,16 @@ void state_transition::export_state_test(const TransactionReceipt& receipt, cons
     auto& jpost = jt["post"][to_test_fork_name(rev)][0];
     jpost["indexes"] = {{"data", 0}, {"gas", 0}, {"value", 0}};
     jpost["hash"] = hex0x(mpt_hash(post.get_accounts()));
-    jpost["logs"] = hex0x(logs_hash(receipt.logs));
+
+    if (holds_alternative<std::error_code>(res))
+    {
+        jpost["expectException"] = std::get<std::error_code>(res).message();
```

Comment:
These are not the right error messages that are normal in tests.
Example of valid one: https://github.com/ethereum/tests/blob/1c23e3c27ac53b794de0844d2d5e19cd2495b9d8/GeneralStateTests/stExample/invalidTr.json#L26

Perhaps some conversion needed similar to https://github.com/ethereum/evmone/blob/9b3ee17421d0f9ed2da555adef7ed81464566e7d/test/unittests/eof_validation.cpp#L14

The good place to find test error messages is probably retesteth config https://github.com/ethereum/retesteth/blob/fa5fe40627b0b4914295d5d0fe120dd0b053cd15/retesteth/configs/clientconfigs/evmone.cpp#L231

Reply:
@gumb0 Thanks for flagging this! So to be sure, what I need to do is:
- PR `retesteth` to add the additional entries
- in this spot, call a new function like `get_tests_error_message(std::error_code)`, instead of just `.message()` which has clauses like:
```
            case INIT_CODE_EMPTY:
                return "TR_InitcodeEmpty";   // <<<<<- one of the new `TR_` entries
```
to translate the code into retesteth-compatible value?

Reply:
yes, sounds good